recipes/shellcheck/bld.bat

```diff
@@ -0,0 +1,28 @@
+set "BINARY_HOME=%PREFIX%\bin"
+set "PACKAGE_HOME=%PREFIX%\share\%PKG_NAME%-%PKG_VERSION%-%PKG_BUILDNUM%"
+set "STACK_ROOT=%PACKAGE_HOME%\stackroot"
+
+mkdir "%BINARY_HOME%"  || goto :error
+mkdir "%PACKAGE_HOME%" || goto :error
+mkdir "%STACK_ROOT%"   || goto :error
+
+stack --local-bin-path "%PREFIX%\bin" ^
+      --extra-include-dirs "%PREFIX%\include" ^
+      --extra-lib-dirs "%PREFIX%\lib" ^
+      --stack-root "%STACK_ROOT%" ^
+      setup ^
+      || goto :error
+stack --local-bin-path "%PREFIX%\bin" ^
+      --extra-include-dirs "%PREFIX%\include" ^
+      --extra-lib-dirs "%PREFIX%\lib" ^
+      --stack-root "%STACK_ROOT%" ^
+      install --ghc-options ^
+        "-optl-pthread -optl-L%PREFIX%\lib -optl-Wl,-rpath,%PREFIX%\lib" ^
```

Comment:
Hmm...what does `-rpath` do on Windows? 🤔

Reply:
I guess that does nothing and can be removed. Will try that.